connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/lifecycle/InboundConnectorManager.java

```diff
@@ -117,10 +118,18 @@ private void deactivateConnector(InboundConnectorProperties properties) {
   private void activateConnector(InboundConnectorProperties newProperties) {
 
     InboundConnectorExecutable executable = connectorFactory.getInstance(newProperties.getType());
+    Consumer<Throwable> cancellationCallback = throwable -> {
+      LOG.error(
+        "Inbound connector failed at correlation point " + newProperties.getCorrelationPointId(),
+        throwable);
+      // TODO: store error for user's convenience
```

Comment:
It would be great if we can convert all todos under one or several tickets to visualize scope for the future work and tracking.

Reply:
This one should cover the todos at least in this class: https://github.com/camunda-community-hub/spring-zeebe/issues/401